frontend/src/components/ViewActivities/activity.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import * as utils from '../Utils/utils';
+import * as DBUTILS from '../../constants/dbconstants'
+import '../../styles/activities.css';
+
+class Activity extends React.Component {
+  /** @inheritdoc */
```

Comment:
Weirdly enough I don't think a lot of React stuff has formal documentation in its source code, the render() method included. So I _think_ it's okay to omit? I am highly unsure about it though and can't find any good answers on it.

Reply:
Better safe than sorry. And if there is documentation, it's inherited.